ibmpisession/ibmpowersession.go

```diff
@@ -80,34 +78,30 @@ func powerJSONConsumer() runtime.Consumer {
 	})
 }
 
-// Create a IBMPISession
-func NewSession(options *PIOptions) (*IBMPISession, error) {
+// Create a IBMPISession using an API Key
```

Comment:
NewSessionApiKey is not specific to API key only. We can also pass BearerToken Authenticator via options here. Hence name change is not needed here.

Reply:
Fixed